source/_patterns/00-base/_mixins.scss

```diff
@@ -1,3 +1,17 @@
+/// Mixins
+
+/// Export Sass Data to JSON in `path/to/export/folder/` folder
+/// @param {String} $filename - ie `mystuff.json`
+/// @param $var - What to turn into JSON
+/// @example scss
+///   @include export-data-to-lib('filename.json', $sass-map);
+@mixin export-data($filename, $var, $options: ()) {
```

Comment:
these are good comments, but please please please get a section in the docs on this + where it's used + why in detail. i guarantee it will make people mad/confused otherwise

Reply:
This is a little shifty as a feature right now (I'm not certain it will survive in its current state), so I'm waiting to spend the time documenting until we get this documenting fonts, typog, grids, etc.